modules/aws/worker/main.tf

```diff
@@ -126,6 +126,6 @@ resource "aws_instance" "worker" {
 
 resource "aws_elb_attachment" "workers" {
   count    = "${length(var.load_balancers) * var.instance_count}"
-  elb      = "${var.load_balancers[count.index % var.instance_count]}"
-  instance = "${aws_instance.worker.*.id[count.index / var.instance_count]}"
+  elb      = "${var.load_balancers[count.index % length(var.load_balancers)]}"
```

Comment:
I expect this would work either way, and `var.instance_count` is shorter and (at least for me) easier to read than `length(var.load_balancers)`.  Is there a reason to switch to using the latter, @yifan-gu?

Reply:
This won't work, e.g.

length(var.load_balancers) == 4
instance_count = 2
count_index = 4*2 = 8

Then, `count.index % var.instance_count` ranges from `[0, 1]`, which won't iterate over all elbs, and  `count.index / var.instance_count` ranges from `[0, 3]` which will be out of boundary.

Reply:
But we can do `instance = [count.index % var.instance_count], elb = [count.index / var.instance_count]`, which will be identical.

Reply:
> But we can do instance = [count.index % var.instance_count], elb = [count.index / var.instance_count], which will be identical.

Done with c42f215 -> f883131.